s3backgrounddelete/s3backgrounddelete/IEMutil.py

```diff
@@ -21,24 +21,27 @@
     eventcode part  005 represents s3background component
     and 0001 repesents connection failures """
 
-import syslog
+import traceback
 
+from cortx.utils.iem_framework import EventMessage
 
 class IEMutil(object):
     severity = ""
     eventCode = ""
     eventstring = ""
+    module = ""
 
     # List of eventcodes
     S3_CONN_FAILURE = "0050010001"
 
     # List of eventstrings
     S3_CONN_FAILURE_STR = "Failed to connect to S3 server. For more information refer the Troubleshooting Guide."
 
-    def __init__(self, loglevel, eventcode, eventstring):
+    def __init__(self, module, loglevel, eventcode, eventstring):
```

Comment:
@nileshgovande 
can we hard code module as S3 BG delete?

Reply:
have default value - OK to pass

Reply:
done